tests/decorators/tests.py

```diff
@@ -474,6 +474,21 @@ def method(self):
         self.assertIsNotNone(func_module)
 
 
+class SyncAndAsyncDecoratorTests(TestCase):
+    """
+    Tests to make sure all builtin decorators declare themselves as sync and
+    async capable.
+    """
+
+    def test_decorators_syanc_and_async_capable(self):
+        decorators = (cache_control,)
+
+        for decorator in decorators:
+            with self.subTest(decorator):
+                self.assertTrue(decorator.sync_capable)
+                self.assertTrue(decorator.async_capable)
```

Comment:
This test has been written in this way so further decorators which are made sync and async capable can be tested by being added to the `decorators` tuple, instead of creating a whole new test per decorator.